BlueprintUI/Sources/BlueprintView/BlueprintView.swift

```diff
@@ -91,6 +91,15 @@ public final class BlueprintView: UIView {
         }
     }
 
+    /// We need to invalidateIntrinsicContentSize when the bounds change for Auto Layout to work correctly.
+    public override var bounds: CGRect {
+        didSet {
+            guard oldValue != bounds else { return }
```

Comment:
Minorest of nits, even though I wrote this code: This could actually check `bounds.size`, not that it matters much.

Reply:
```suggestion
            guard oldValue.size != bounds.size else { return }
```